templates/monogame/layout/_master.tmpl

```diff
@@ -104,67 +104,118 @@
+              <li class="nav-item mb-2"><a href="/api/index.html" class="footer-link p-0 text-body-secondary">API Reference</a></li>
+              <li class="nav-item mb-2"><a href="/showcase.html" class="footer-link p-0 text-body-secondary">Showcase</a></li>
+              <li class="nav-item mb-2"><a href="/about.html" class="footer-link p-0 text-body-secondary">About</a></li>
+              <li class="nav-item mb-2"><a href="/foundation/index.html" class="footer-link p-0 text-body-secondary">MonoGame Foundation Bylaws</a></li>
+            </ul>
+          </div>
+          <div class="col-6 col-md-2 mb-3">
+            <h5>Get Involved</h5>
+            <ul class="nav flex-column">
+              <li class="nav-item mb-2"><a href="https://github.com/MonoGame/MonoGame" class="footer-link p-0 text-body-secondary">Source Code</a></li>
+              <li class="nav-item mb-2"><a href="https://github.com/MonoGame/monogame.github.io/blob/main/CONTRIBUTING.md" class="footer-link p-0 text-body-secondary">Update Documentation</a></li>
+              <li class="nav-item mb-2"><a href="https://github.com/MonoGame/MonoGame/issues/new/choose" class="footer-link p-0 text-body-secondary">Report Issues</a></li>
+              <li class="nav-item mb-2"><a href="https://www.patreon.com/MonoGame" class="footer-link p-0 text-body-secondary">Patreon</a></li>
+            </ul>
+          </div>
+          <div class="col-6 col-md-2 mb-3">
+            <h5>Community</h5>
+              <ul class="nav flex-column">
+                <li class="nav-item mb-2"><a href="https://community.monogame.net" class="footer-link p-0 text-body-secondary">Forums</a></li>
+                <li class="nav-item mb-2"><a href="https://discord.gg/monogame" class="footer-link p-0 text-body-secondary">Discord</a></li>
+                <li class="nav-item mb-2"><a href="https://twitter.com/MonoGameTeam" class="footer-link p-0 text-body-secondary">Twitter</a></li>
+                <li class="nav-item mb-2"><a href="https://www.youtube.com/MonoGame" class="footer-link p-0 text-body-secondary">YouTube</a></li>
+                <li class="nav-item mb-2"><a href="https://www.reddit.com/r/monogame/" class="footer-link p-0 text-body-secondary">Reddit</a></li>
+                <li class="nav-item mb-2"><a href="https://www.facebook.com/monogamecommunity" class="footer-link p-0 text-body-secondary">Facebook</a></li>
+              </ul>
+          </div>
+        </div>
+        <div class="d-flex flex-column flex-sm-row justify-content-between py-4 my-4">
+          <div>
+            {{_appFooter}} <br/>
+            {{#_overrideDocFx}}
+            Home page designed with ❤ by <a href="https://github.com/AristurtleDev">Christopher Whitley (AristurlteDev)</a> with feedback from community members<br/>
+            Code, Documentation, and Report Issues icons created by community member TankStory
```

Comment:
Not trying to properly attribute you on this work @AristurtleDev, but can we find a simpler way to attribute people than this?

If 2 or 3 more people start contributing do we then list all of them?  My concern is the footer getting huge with lots of credits if we set this precedent of listing people and specifically what they contributed.

Maybe we do something much simpler like this:

Designed with ❤ by [MonoGame community](https://github.com/MonoGame/monogame.github.io/graphs/contributors).

The link there being to the contributor graph for the website.  This then has everyone listed that contributed to the website with specifically how much they contributed to it.

We can then use this across all the footers on the website.

Reply:
Also wouldn't mind:

Designed with ❤ by (AristurtleDev)[https://github.com/AristurtleDev] and the [MonoGame Community](https://github.com/MonoGame/monogame.github.io/graphs/contributors).

Just want to keep it from getting much bigger over time.

Reply:
I like the MonoGame Community idea. I felt weird crediting myself, but it was a common thing to do, but just dropping it to community with link to the contributors is a great idea. Updated to this in https://github.com/MonoGame/monogame.github.io/pull/19/commits/d24ce652460368874111abeb1291cb179ac2ba17